liquibase-standard/src/main/java/liquibase/util/StandardSqlParser.java

```diff
@@ -20,7 +20,7 @@ public StringClauses parse(String sqlBlock, boolean preserveWhitespace, boolean
         SimpleSqlGrammar t = new SimpleSqlGrammar(tokenManager);
         try {
             Token token = t.getNextToken();
-            while (!"".equals(token.toString())) {
+            while (!token.toString().isEmpty()) {
```

Comment:
I think we can have a null value here.

Reply:
I think in this particular case we should check `token != null && !token.toString().isEmpty()`, because if we use `StringUitl.isNotEmpty(token)` we would allow null and then when processing the token get an NPE. Do you agree?

Reply:
Makes sense!